client/src/context/dashboard/dashboard.provider.js

```diff
@@ -54,5 +54,5 @@ const DashboardProvider = (props) => {
     )
 }
 
-export {DashboardProvider, DashboardContext};
+export {DashboardProvider, DashboardContext, useDashboard};
```

Comment:
As a readability preference, One may directly use DashboardContext to extract context instead of a separate useDashboard method